src/mobile/apps/auth/test/client.test.coffee

```diff
@@ -98,6 +98,7 @@ describe 'Sign up (with email) client-side code', ->
 describe 'Login view', ->
 
   beforeEach (done) ->
+    global.location = { search: '' }
```

Comment:
Why not `global.location.search = ''`? In this case the entire `search` object is overwritten.